eclair-rpc-test/src/test/scala/org/bitcoins/eclair/rpc/EclairRpcClientTest.scala

```diff
@@ -742,6 +679,120 @@ class EclairRpcClientTest extends AsyncFlatSpec with BeforeAndAfterAll {
+        fourthClient
+          .receive("foo")
+          .flatMap { invoice =>
+            firstClientF.flatMap(_.findRoute(invoice, None))
+          }
+          .map(route => route.length == 4)
+          .recover {
+            case err: RuntimeException
+                if err.getMessage.contains("route not found") =>
+              false
+          }
+      }
+
+    }
+
+    // Eclair is a bit slow in propagating channel changes
+    AsyncUtil.awaitConditionF(hasRoute, duration = 10000.millis, maxTries = 20).map(_ => succeed)
+  }
+
+
+*/
+
+  it should "update the relay fee of a channel" in {
+    val channelAndFeeF = for {
+      channel <- openAndConfirmChannel(clientF, otherClientF)
+      feeOpt <- clientF.flatMap(_.channel(channel).map(_.feeBaseMsat))
+    } yield {
+      assert(feeOpt.isDefined)
+      assert(feeOpt.get > MilliSatoshis.zero)
+      (channel, feeOpt.get)
+    }
+
+    for {
+      (channel, oldFee) <- channelAndFeeF
+      _ <- clientF.flatMap(
+        _.updateRelayFee(channel, MilliSatoshis(oldFee.toLong * 2), 1))
+      newFeeOpt <- clientF.flatMap(_.channel(channel).map(_.feeBaseMsat))
+    } yield {
+      assert(newFeeOpt.isDefined)
+      assert(newFeeOpt.get == MilliSatoshis(oldFee.toLong * 2))
+    }
+  }
+
+  it should "update the relay fee of a channel with short channel id" in {
+    val channelAndFeeF = for {
+      channelId <- openAndConfirmChannel(clientF, otherClientF)
+      client <- clientF
+      channel <- client.channel(channelId)
+    } yield {
+      assert(channel.feeBaseMsat.isDefined)
+      assert(channel.feeBaseMsat.get > MilliSatoshis.zero)
+      (channel.channelId, channel.shortChannelId, channel.feeBaseMsat.get)
+    }
+
+    for {
+      (channelId, shortChannelId, oldFee) <- channelAndFeeF
+      _ <- clientF.flatMap(
+        _.updateRelayFee(shortChannelId, MilliSatoshis(oldFee.toLong * 4), 1))
+      newFeeOpt <- clientF.flatMap(_.channel(channelId).map(_.feeBaseMsat))
+    } yield {
+      assert(newFeeOpt.isDefined)
+      assert(newFeeOpt.get == MilliSatoshis(oldFee.toLong * 4))
+    }
+  }
+
+  it should "get channels" in {
+    clientF.flatMap(_.channels().flatMap(_ =>  succeed))
+  }
+
+  it should "get all channels" in {
+    clientF.flatMap(_.allChannels().flatMap(_ => succeed))
+  }
+
+  it should "get all channel updates" in {
+    clientF.flatMap(_.allUpdates().flatMap { _ =>
+      succeed
+    })
+  }
+
+  it should "get updates for a single node" in {
+    for {
+      client <- clientF
+      nodeInfo <- client.getInfo
+      _ <- client.allUpdates(nodeInfo.nodeId)
+    } yield {
+      succeed
+    }
+  }
+
+  it should "get all nodes" in {
+    clientF.flatMap(_.allNodes().flatMap(_ => succeed))
+  }
+
+  it should "get a route to a node ID" in {
+    val hasRoute = () => {
+      fourthClientF
+        .flatMap(_.getInfo)
+        .flatMap(info => firstClientF.flatMap(_.findRoute(info.nodeId, MilliSatoshis(100))))
```

Comment:
So this is not a big deal at all, just thought I should say it so you're aware of it: It's possible to write money like this:

```scala
100.msat == MilliSatoshis(100)
```

The extension method is provided by the package object in `org.bitcoins.core.protocol.ln.currency`